src/ds/d500/hw_monitor_extended_buffers.cpp

```diff
@@ -67,13 +67,10 @@ namespace librealsense
 
         if (recv_msg_length > HW_MONITOR_BUFFER_SIZE)
         {
-            // currently we assume HKR sends only the table size when sending 0/0 chunks command above.
-            // in the future the FW should send chunk 1 in 0/0 command and then we can use it here.
-            // meaning remove the 'clear' and start the loop from 1
-
-            recv_msg.clear();
```

Comment:
you removed the clear command
So the received message on the second call with be appended after the first one - according to the title of this PR, it seems this was your aim, but please explain.

Reply:
Yes, as you know, we have 2 use cases.
* HMC < 1000 bytes
  _We append the full message to recv_msg and skip the next loop_
* HMC > 1000 bytes
* _We append the first chunk message to recv_msg, then step into the loop and append chunk 1 - (num_of_chunks - 1) _

Sounds OK?